[2025-03-12T10:09:47.953+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T10:09:48.004+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_product_analysis.customers manual__2025-03-12T10:09:42.150569+00:00 [queued]>
[2025-03-12T10:09:48.034+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_product_analysis.customers manual__2025-03-12T10:09:42.150569+00:00 [queued]>
[2025-03-12T10:09:48.035+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T10:09:48.081+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): customers> on 2025-03-12 10:09:42.150569+00:00
[2025-03-12T10:09:48.092+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=614) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T10:09:48.096+0000] {standard_task_runner.py:72} INFO - Started process 622 to run task
[2025-03-12T10:09:48.098+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'daily_product_analysis', 'customers', 'manual__2025-03-12T10:09:42.150569+00:00', '--job-id', '463', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpomudc4jd']
[2025-03-12T10:09:48.100+0000] {standard_task_runner.py:105} INFO - Job 463: Subtask customers
[2025-03-12T10:09:48.198+0000] {task_command.py:467} INFO - Running <TaskInstance: daily_product_analysis.customers manual__2025-03-12T10:09:42.150569+00:00 [running]> on host b404e9833d11
[2025-03-12T10:09:48.388+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_product_analysis' AIRFLOW_CTX_TASK_ID='customers' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T10:09:42.150569+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T10:09:42.150569+00:00'
[2025-03-12T10:09:48.391+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T10:09:48.392+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T10:09:48.392+0000] {logging_mixin.py:190} INFO - Current task name:customers state:running start_date:2025-03-12 10:09:48.005860+00:00
[2025-03-12T10:09:48.393+0000] {logging_mixin.py:190} INFO - Dag name:daily_product_analysis and current dag run status:running
[2025-03-12T10:09:48.394+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T10:09:49.268+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-03-12T10:09:49.609+0000] {logging_mixin.py:190} INFO - Validation successful. Found 1445 customer records.
[2025-03-12T10:09:49.610+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T10:09:49.623+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T10:09:49.624+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=daily_product_analysis, task_id=customers, run_id=manual__2025-03-12T10:09:42.150569+00:00, execution_date=20250312T100942, start_date=20250312T100948, end_date=20250312T100949
[2025-03-12T10:09:49.660+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T10:09:49.661+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T10:09:49.662+0000] {logging_mixin.py:190} INFO - Dag name:daily_product_analysis queued_at:2025-03-12 10:09:42.168285+00:00
[2025-03-12T10:09:49.663+0000] {logging_mixin.py:190} INFO - Task hostname:b404e9833d11 operator:PythonOperator
[2025-03-12T10:09:49.688+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T10:09:49.735+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T10:09:49.738+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
